sdk/src/AbstractMain.cpp

```diff
@@ -199,13 +199,15 @@ int AbstractMain::run(int in_argc, char** in_argv)
    addLogDestination(
       std::shared_ptr<ILogDestination>(
          new SyslogDestination(
+            "",
             LogLevel::INFO,
-            getProgramId())));
+            LogMessageFormatType::PRETTY,
+            getProgramId().c_str())));
 
    logging::logInfoMessage("Starting plugin...");
 
    // Turn on stderr logging while options are parsed.
-   std::shared_ptr<ILogDestination> stderrLogDest(new StderrLogDestination(LogLevel::INFO));
+   std::shared_ptr<ILogDestination> stderrLogDest(new StderrLogDestination("",LogLevel::INFO, LogMessageFormatType::PRETTY));
```

Comment:
```suggestion
   std::shared_ptr<ILogDestination> stderrLogDest(new StderrLogDestination(ore::system::generateShortenedUuid(),LogLevel::INFO, LogMessageFormatType::PRETTY));
```

Log destinations should be given a unique ID so they don't collide.